src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,117 @@
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\FormatOnType",
+          "bitmask": 1
+        }
+      }
+    },
+    "textEditor.razor.advanced.autoClosingTags": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_AutoClosingTagsDisplayName%",
+      "description": "%Setting_AutoClosingTagsDescription%",
+      "migration": {
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\AutoClosingTags",
+          "bitmask": 1
+        }
+      }
+    },
+    "textEditor.razor.advanced.autoInsertAttributeQuotes": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_AutoInsertAttributeQuotesDisplayName%",
+      "description": "%Setting_AutoInsertAttributeQuotesDescription%",
+      "migration": {
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\AutoInsertAttributeQuotes",
+          "bitmask": 1
+        }
+      }
+    },
+    "textEditor.razor.advanced.colorBackground": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_ColorBackgroundDisplayName%",
+      "description": "%Setting_ColorBackgroundDescription%",
+      "migration": {
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\ColorBackground",
+          "bitmask": 1
+        }
+      }
+    },
+    "textEditor.razor.advanced.commitElementsWithSpace": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_CommitElementsWithSpaceDescription%",
+      "description": "%Setting_CommitElementsWithSpaceDisplayName%",
+      "migration": {
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\CommitElementsWithSpace",
+          "bitmask": 1
+        }
+      }
+    },
+    "textEditor.razor.advanced.snippets": {
+      "type": "string",
+      "default": "all",
+      "enum": [
+        "all",
+        "custom",
+        "none"
+      ],
+      "enumItemLabels": [
+        "%Setting_SnippetsAll%",
+        "%Setting_SnippetsCustom%",
+        "%Setting_SnippetsNone%"
+      ],
+      "title": "%Setting_CommitElementsWithSpaceDescription%",
+      "description": "%Setting_CommitElementsWithSpaceDisplayName%",
+      "migration": {
+        "enumIntegerToString": {
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\Snippets"
+          },
+          "map": [
+            {
+              "result": "all",
+              "match": 0
+            },
+            {
+              "result": "custom",
+              "match": 1
+            },
+            {
+              "result": "none",
+              "match": 2
+            }
+          ]
+        }
+      }
+    }
+  },
+  "categories": {
+    "razor": {
+      "title": "%110%" // Razor (ASP.NET Core)
+    },
+    "razor.advanced": {
+      "title": "%1050%", // Advanced
+      "order": 0,
```

Comment:
The order value for pages migrated from Options should match the one specified by the Options page.  In this case, your Options page didn't specify a sort order, so you should omit it here as well.